backend/density-scrape.mjs

```diff
@@ -1,28 +1,28 @@
-import puppeteer from 'puppeteer';
+import puppeteer from "puppeteer";
 
 async function scrape(food) {
-	while( food.indexOf(" ") != -1){
-		food = food.replace(" ", "-")
-	}
-	console.log("food is now " + food)
-    const browser = await puppeteer.launch();
-    const page = await browser.newPage();
-    await page.goto("https://www.aqua-calc.com/page/density-table/substance/"+food);
```

Comment:
Kind of weird that github considered this a merge conflict, the code stays the same